packages/neuron-wallet/src/utils/ckb-rpc.ts

```diff
@@ -131,6 +131,12 @@ export class FullCKBRPC extends CKBRPC {
   }
 }
 
+class Method extends SdkRpcMethod {
```

Comment:
Maybe `SdkRpcMethod` can be used directly

Reply:
the super bond a `rpcConfig` as `super(node, options, rpcConfig)`, so we use an extended class here